tests/estimation/test_get_estimation.py

```diff
@@ -7,12 +7,18 @@
 The test xfails for any other wierd behavior.
 
 We pinpoint that :
-- DML_huber is not working for dim_x=1, RRuntimeError is raised
-- multiply_robust methods return some NaN effects (half not implemented)
-- t.ravel() and y.ravel() are necessary to get IPW proper behavior
+- DML_huber is not working for dim_x=1, RRuntimeError is raised (#15)
```

Comment:
Are these things you want to address in this PR ? If so, you should put that as a comment, not leave it in the code.

Reply:
Not exactly, I wanted to make a script docstrings but I ended up writing issues and PR stuff in it...
I'll make the docstrings cleaner

Reply:
done, c8bb4ba